api/src/main/java/org/openmrs/module/appointments/conflicts/impl/AppointmentServiceUnavailabilityConflict.java

```diff
@@ -19,26 +19,27 @@
 
 public class AppointmentServiceUnavailabilityConflict implements AppointmentConflictType {
 
-    private final SimpleDateFormat DayFormat = new SimpleDateFormat("EEEE");
+    private final static String DAY_OF_WEEK_PATTERN = "EEEE";
+    private final SimpleDateFormat DayFormat = new SimpleDateFormat(DAY_OF_WEEK_PATTERN);
 
     @Override
-    public String getType() {
-        return SERVICE_UNAVAILABLE.name();
+    public Enum getType() {
+        return SERVICE_UNAVAILABLE;
     }
 
     @Override
     public List<Appointment> getAppointmentConflicts(List<Appointment> appointments) {
         List<Appointment> conflictingAppointments = new ArrayList<>();
         for (Appointment appointment : appointments) {
             AppointmentServiceDefinition appointmentServiceDefinition = appointment.getService();
-            Appointment conflict = checkConflicts(appointment, appointmentServiceDefinition);
+            Appointment conflict = getConflictingAppointment(appointment, appointmentServiceDefinition);
```

Comment:
Return boolean and keep `checkConflicts`

Reply:
done.